app/src/main/java/net/squanchy/injection/ContextModule.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.injection;
+
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class ContextModule {
```

Comment:
Unfortunately ActivityContextModule needs an Activity object to be initialized, even though it just provides a Context (not sure if there's a reason for that). So it can't be used in components that are not dependent on an Activity (like the ApplicationComponent).
Is it ok if I just squash the ActivityContextModule and migrate its dependents to use a ContextModule? They surely don't need an activity since the provide method is
```java
@Provides
    Context activityContext() {
        return activity;
    }
```